protocol/contracts/beanstalk/AppStorage.sol

```diff
@@ -640,4 +641,5 @@ struct AppStorage {
     mapping(IERC20 => uint256) internalTokenBalanceTotal;
```

Comment:
SopWell should be removed in storage.

Reply:
Renamed, but I see this has various other places it needs to be updated, will work on those. https://github.com/BeanstalkFarms/Beanstalk/commit/0b27aa1554731311cb9d4834f9d18f3baa895682